### DISCUSSION TITLE
Concepts and difference of Neighbor distance, Mean Interbuilding Distance, and Building adjacency

### DISCUSSION THREAD

**Discussion by cdkang** [pysal/momepy#366] [Q&A] [answered]
Hello, 

Now I utilized momepy to capture urban form of Seoul. 

The description of Neighbor distance, Mean Interbuilding Distance, and Building adjacency looks similar for me. 

Could you give me more detail description on the difference of Neighbor distance, Mean Interbuilding Distance, and Building adjacency? 

Sincerely,

**Comment by martinfleis** [comment, marked as answer]
Hey, they are a bit similar but not the same. 

Neighbour distance and mean inter building distance are capturing the same but on a different scale. 

- Neighbour distance measures a mean distance to immediate neighbours of each buildings given a contiguity weights matrix. Let's say that a building A neighbours buildings B and C, it measures a mean of {A-B, A-C} distances.
- Mean inter building distance measures a mean distance in between all immediate neighbours within a context defined by another weights matrix, usually of a higher order. Using the example above, the result is a mean of {A-B, A-C, B-C} given B and C are neighbouring each other.
- Building adjacency is a bit different as it does not measure distance between buildings but their topology and tries to assess the degree to which buildings are directly touching each other, forming a single composite.

**Comment by cdkang** [comment]
Thanks a lot. Your description is very helpful.
